.buildkite/steps/tests.sh

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 go version
 echo arch is "$(uname -m)"
 
-GO111MODULE=off go get gotest.tools/gotestsum
+go install gotest.tools/gotestsum@v1.8.0
```

Comment:
Am I correct in thinking image-builder already installs this, but installing it again is probably a reasonably fast no-op?

Reply:
Yep! pretty much. Also, for linux, we test in a go container, which doesn't have this installed. The windows images do; but we want to run the same script on each, ideally

Reply:
Ah yeah!